src/main/resources/db/changelog/202410290000000-drop-fk-cp-owner.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.19.xsd">
+
+    <changeSet id="202410290000000-1" author="sbakaj">
+        <dropForeignKeyConstraint baseTableName="cp2_owner_products" constraintName="cp2_owner_products_fk1"/>
```

Comment:
Add checks to make sure these FKs still exist before we do the drop

Reply:
done